### DISCUSSION TITLE
Error during dynamic loading module

### DISCUSSION THREAD

**Discussion by kyooi93** [PrismLibrary/Prism#2407] [WPF]
![image](https://user-images.githubusercontent.com/71051053/115512728-dc5fa200-a2b4-11eb-80de-ddd33daa6a71.png)

Hello all,

I'm using Entity Framework Core 3.1.14. When dynamic loading the module , there come out the error. Please Help . Thanks alot

Best Regards,
KY Ooi

**Comment by brianlagunas** [comment]
Please do not post the same thing in both issues and discussions.  Reference #2408

**Reply by kyooi93** [reply]
okay.noted
